exercises/D-predicates/exercise.js

```diff
@@ -2,14 +2,22 @@
   Predicates
   ---------------------------------
   Write a predicate to predicates
-  The variables should have values that match the expected results.
+  The variables should have value             
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  return typeof value === "number";
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
+  if (number > 0 && number < 10){
```

Comment:
Something to keep in mind is that this statement: `number > 0 && number < 10` is going to return a boolean value(true or false), and you can shorten your code by writing something like:
```suggestion
  function isBetweenZeroAnd10(number) {
  return number > 0 && number < 10;
  }
```
This is just something to keep in mind when writing code. 
Hope this helps.